### PR TITLE
fix: hide contact gallery CTA for institution partners

### PR DESCRIPTION
**Pull request by dblandin**
The type of this PR is: **Fix**

This PR solves [FX-4327]

### Description

Hides the "Contact Gallery" button in the partner info component within the artwork page.

---

### Screenshots

**Before**

Screen Shot 2022-10-05 at 18 13 44

**After**

Screen Shot 2022-10-05 at 18 11 59

[FX-4327]: https://artsyproduct.atlassian.net/browse/FX-4327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dblandin** [comment]
Closing this for now as we continue chatting in [Slack](https://artsy.slack.com/archives/C9SATFLUU/p1664987682539129?thread_ts=1664896451.259369&cid=C9SATFLUU)/Jira.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx [hunk 1]
```diff
@@ -82,16 +82,19 @@ export const ArtworkSidebarPartnerInfo: FC<ArtworkSidebarPartnerInfoProps> = ({
           )}
         </PartnerContainer>
 
-        {isCBNEnabled && !isInquireable && !is_in_auction && (
```
Comment (anandaroop) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx:85]: Interesting that the `isInquireable` check was not sufficient for hiding the contact button 🤔 

I believe that maps back to this in Gravity:

https://github.com/artsy/gravity/blob/ffd6b8cb3e17d0258c732d88a4a47d71956e3bbe/app/models/domain/artwork.rb#L666-L671

```ruby
  def inquireable?
    return false if offer || ecommerce
    return true if partner.try(:inquireable?)

    !!(forsale? || at_auction?)
  end
```

The partner in question:

```ruby
Partner.find('walters-art-museum').inquireable?
# => false
```

Comment (anandaroop) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx:85]: Oh wait, that check is a negation? `!isInquireable`

Comment (dblandin) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx:85]: Yeah, it's definitely convoluted at the moment. I think the intent was to show this button only if we're not exposing Contact Gallery as a primary commercial CTA.

**Pull request closed without merging**